integration/tests/02-create-package-repository.spec.js

```diff
@@ -0,0 +1,36 @@
+const { test, expect } = require("@playwright/test");
+const { KubeappsLogin } = require("./utils/kubeapps-login");
+const utils = require("./utils/util-functions");
+
+test("Create a new package repository successfully", async ({ page }) => {
+  test.setTimeout(60000);
+
+  // Log in
+  const k = new KubeappsLogin(page);
+  await k.doLogin("kubeapps-operator@example.com", "password", process.env.ADMIN_TOKEN);
+
+  // Go to repos page
+  await page.goto(utils.getUrl("/#/c/default/ns/kubeapps/config/repos"));
+
+  // Add new repo
+  console.log('Creating repository "my-repo"');
+  await page.click('cds-button:has-text("Add App Repository")');
+  await page.type("input#kubeapps-repo-name", "my-repo");
+  await page.type("input#kubeapps-repo-url", "https://charts.gitlab.io/");
+  await page.click('cds-button:has-text("Install Repo")');
+
+  // Wait for new packages to be indexed
+  await page.waitForTimeout(5000);
+
+  // Check if packages show up in catalog
+  // await page.reload({ waitUntil: "networkidle" });
+  await page.click('a:has-text("my-repo")');
+  await page.waitForSelector('css=.catalog-container .card-title >> text="gitlab-runner"');
+
+  // Clean up
+  const axInstance = await utils.getAxiosInstance(page);
+  const response = await axInstance.delete(
+    "/api/v1/clusters/default/namespaces/kubeapps/apprepositories/my-repo",
```

Comment:
Hmm, unless you haven't merged main lately, this should not succeed, since the k8s api is no longer being proxied (by default)?

Reply:
This works due to the CI pipeline running with Operators and k8s APIs enabled.
We will have to do the delete operation using the UI. I added some clean up actions to keep the cluster freed from having many workloads due to limited resources.

Reply:
> This works due to the CI pipeline running with Operators and k8s APIs enabled.

Yeah, I just meant that if you merge main, the `e2e-test.sh` script should no longer do that, but rather run all the main tests without the k8s APIs enabled, then install the operator support, upgrade kubeapps and run just the operator test.

> We will have to do the delete operation using the UI. I added some clean up actions to keep the cluster freed from having many workloads due to limited resources.

Yep, ok.

Reply:
Fixed that problem, now operators test is run correctly after all main tests.